File: python/grammer/07_set.py
"""7) 집합(set): 중복 제거와 빠른 포함 검사."""

nums = [1, 1, 2, 3, 3, 4]
unique_nums = set(nums)

print("원본 리스트:", nums)
print("중복 제거 결과:", unique_nums)
print("3이 있는가?:", 3 in unique_nums)
print("10이 있는가?:", 10 in unique_nums)

File: javascript/grammer/07_set.js
const nums = [1, 1, 2, 3, 3, 4];
const uniqueNums = new Set(nums);

console.log('원본 배열:', nums);
console.log('중복 제거 결과:', [...uniqueNums]);
console.log('3이 있는가?:', uniqueNums.has(3));
console.log('10이 있는가?:', uniqueNums.has(10));
